scripts/perf/perf_config.properties

```diff
@@ -0,0 +1,6 @@
+
+REMOTE_SERVER_IP=127.0.0.1
+REMOTE_SPAMMER_IP=127.0.0.1
+REMOTE_SECOND_SERVER_IP=127.0.0.1
```

Comment:
second server is not required now. right? use the spammer env to start the other node